test/external-e2e/manifest/testdriver.yaml

```diff
@@ -19,5 +19,8 @@ DriverInfo:
         volumeLimits: true
         snapshotDataSource: true
         pvcDataSource: true
+        singleNodeVolume: true
```

Comment:
you may revert this to only upgrade ginko v2 first

Reply:
nvm, there is only one external test case failure in this PR, let's rerun the tests first.